client/src/components/shared/PageHeadElement.tsx

```diff
@@ -1,12 +1,38 @@
 import { app, subcommunities } from '@/helpers/common';
 import Head from 'next/head';
 
-export default function PageHeadElement({ landingPage, subcommunityName, pageName }: { landingPage?: boolean; subcommunityName: string; pageName: string }) {
+export default function PageHeadElement({
+  landingPage,
+  subcommunityName,
+  pageName,
+}: {
+  landingPage?: boolean;
+  subcommunityName: string;
+  pageName: string;
+}) {
   const appName = landingPage ? app.name : subcommunityName;
   const PageTitle = `${appName} | ${pageName}`;
   return (
     <Head>
-        <title>{ PageTitle }</title>
+      <link
+        rel="apple-touch-icon"
+        sizes="180x180"
+        href="/apple-touch-icon.png"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="32x32"
+        href="/favicon-32x32.png"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="16x16"
+        href="/favicon-16x16.png"
+      />
+      <link rel="manifest" href="/site.webmanifest" />
+      <title>{PageTitle}</title>
     </Head>
   );
-};
+}
```

Comment:
**_Review comment 1_**
@AbrahamAkp please PageHeadElement file contain only the changes you are making. Restore all formatting to what was there previously.

Reply:
Sorry i don't understand. the <title>{PageTitle}</title> which is the previous code is still in the component.

Reply:
The latter part - "Restore all formatting to what was there previously". This is the problem.

Reply:
Only the changes you are or should be making is what we are interested in. Any other thing should not change in the file.

Reply:
Oh I get now. The issue is the prettier I am using in my VSCode. Saving will definately reformat the code. what do you suggest? Maybe we can setup a global formatter and add to the guidelines as well. I will uninstall my pretttier so it doesn't format it.

Reply:
I have the idea. We will set it up. For now, you can just do away with your formatter anytime you are making changes to this repository.

Reply:
ok

Reply:
PR updated.